### ISSUE TITLE
Shantae Half-Genie Hero Translation - C2-12828-1 Error

### ISSUE THREAD

**Issue by Rameron** [dots-tb/rePatch-reDux0#41]
Hello!

**Describe the bug**
Translated DLC doesn't work in rePatch reDux0 v3.0, but it works in rePatch reDux0 2.71 (ALPHA).

**To Reproduce**
Problem is with game Shantae Half-Genie Hero [PCSE00950] (Patch 1.05 + DLCs) + RePatch translation: https://www.mediafire.com/file/lijrg777n29dzjo/Shantae_Half-Genie_Hero_%5BPCSE00950%5D_%5BNoNpDrm%5D_%5BRus%5D_%5BUPD_14-12-2018%5D.zip/file

**Additional context**
Without ReAddcont folder game successfully started with the translation of the main game.

**Get logging information:**
logger.txt:

loaded ur0:LOLIcon/main/config.bin
forcing reset
# Library not found: [SceLibc, ver=1]
[VitaShellPatch             ]:Starting...  OK        350usec
loaded ur0:LOLIcon/VITASHELL/config.bin
forcing reset
loaded ur0:LOLIcon/main/config.bin
forcing reset
loaded ur0:LOLIcon/PCSE00950/config.bin
forcing reset
[SceCommonDialog] Starting Init(), type 10
[SceCommonDialog] Term() completed, type 10
3:0x9cd098e26c7c132a:0x8c2ff5fd:[coredump] start ID=PCSE00950
loaded ur0:LOLIcon/main/config.bin
forcing reset
3:0x9cd098e2e133745a:0x8c2ff5fd:[coredump] done
# Library not found: [SceLibc, ver=1]
[VitaShellPatch             ]:Starting...  OK        347usec
loaded ur0:LOLIcon/VITASHELL/config.bin
forcing reset
loaded ur0:LOLIcon/main/config.bin
forcing reset
# Library not found: [SceLibc, ver=1]
this is a testloaded ur0:LOLIcon/SHIP00001/config.bin
forcing reset

Thanks.

**Comment by dots-tb** [comment]
NVM. Are you using it with a compat pack and/or are you using any other plugins?

**Comment by Rameron** [comment]
No, I'm using the console with 3.68 firmware and those plugins:

config.txt
***KERNEL**
#storagemgr.skprx : v3.0
ur0:tai/storagemgr.skprx

#nonpdrm.skprx : v1.2
ur0:tai/nonpdrm.skprx

#nopsmdrm.skprx : v1.3
ur0:tai/nopsmdrm.skprx

#repatch_redux0_v3_0.skprx : v3.0
ur0:tai/repatch_redux0_v3_0.skprx

#vitabright.skprx : v0.9
ur0:tai/vitabright.skprx

#kuio.skprx : v1.2 (Rinnegatamante)
ur0:tai/kuio.skprx

#adrenaline_kernel.skprx v6.7
ux:app/PSPEMUCFW/sce_module/adrenaline_kernel.skprx

#ioPlus.skprx: v3.2
ur0:tai/ioPlus.skprx 

#ShipLog
ur0:tai/backdoor_exe.skprx

***main**
ur0:tai/henkaku.suprx

#nolockscreen_v2.suprx : v2.0
ur0:tai/nolockscreen_v2.suprx

#shellbat.suprx : r9
ur0:tai/shellbat.suprx

#download_enabler.suprx : v1.0
ur0:tai/download_enabler.suprx

#pngshot.suprx : v1.2
ur0:tai/pngshot.suprx

***NPXS10015**
ur0:tai/henkaku.suprx

***NPXS10016**
ur0:tai/henkaku.suprx

***ALL**
#VitaGrafix.suprx : v3.2
ur0:tai/VitaGrafix.suprx

***!PSPEMUCFW**
***KERNEL** 
#LOLIcon.skprx : v1.0.1.1 HOTFIX A
ur0:tai/LOLIcon.skprx

**Comment by dots-tb** [comment]
Do you know if disabling VitaGrafix and kuio prevents a crash? And maybe if you need to, disable ioPlus too.

**Comment by Yoti** [comment]
Same for me. Crash on v3.0 and no problems under v2.71. Do we have any debug build to get logs?
